docs/gcloud-auth.rst

```diff
@@ -0,0 +1,264 @@
+
+   # Create two different topics.
+   pubsub.Topic('topic1', project='project1', connection=connection1).create()
+   pubsub.Topic('topic2', project='project2', connection=connection2).create()
+
+If you have one "main" set of credentials
+and others are "one-offs" for special cases,
+you can avoid all this typing
+by using the default connection for the main set,
+and separate connections for the others.
+
+Using the same example as before,
+and assuming ``'key1.json'`` is the "main" set of credentials,
+here's what that same code might look like.
+
+.. code-block:: python
+
+   from gcloud import pubsub
+   from gcloud.credentials import get_for_service_account_json
+
+   credentials2 = get_for_service_account_json('key2.json')
+   connection2 = pubsub.Connection(credentials=credentials2)
+
+   pubsub.Topic('topic1').create()
+   pubsub.Topic('topic2', project='project2', connection=connection2).create()
+
+===============
+Troubleshooting
+===============
+
+--------------------------------------------------
+You keep mentioning key.json... How do I get that?
+--------------------------------------------------
+
+When we say ``key.json``
+we're referring to what we call a "Service Account".
+These are sort of like passwords
+which let you access the different cloud services.
+
+Service Accounts are pretty easy to create,
+just head over to the `Credentials`_ page
+in the `Google Developers Console`_,
+create a new Client ID,
+and generate a new JSON key.
+
+If you're a bit confused,
+take a look at the step by step guide here:
+https://cloud.google.com/storage/docs/authentication#service_accounts
+
+----------------------------------------------------------------
+I'm running in Compute Engine, but can't access Datastore. Help?
+----------------------------------------------------------------
+
+When you create your Compute Engine instance,
+you need to specify which services this particular instance has access to
+(you might not want all of your machines to access Datastore for example).
+
+To do this,
+click on "Show advanced options" on the "New Instance" page
+(at the top of the screen, towards the right),
+scroll down to the section called "Project Access",
+and set the appropriate access
+for the service you want your VM to have
+(ie, Datastore -> Enabled).
+
+.. note::
+
+   Unfortunately,
+   you can't change these settings after the VM is created. :(
+   Hopefully Google will change this in the future.
+
+-------------------------------
+I'm still having trouble. Help?
+-------------------------------
+
+If you're having trouble authenticating,
+feel free to open a `Github Issue`_ on this project
+and hopefully we can help
+(and add the answer to your question here).
+
+.. _Google Developers Console: https://console.developers.google.com/project
+.. _Credentials: https://pantheon.corp.google.com/project/_/apiui/credential
+.. _Github Issue: https://github.com/GoogleCloudPlatform/gcloud-python/issues/new?title=I+need+help+with+auth!
```

Comment:
Sure - updated.

(I couldn't find a way... fiddled with the URL and looked in the docs... Maybe we can ask?)